package.json

```diff
@@ -1,10 +1,13 @@
 {
   "name": "eslint-plugin-ts-paths-import-prefer",
   "version": "0.0.0",
-  "main": "index.js",
+  "main": "dist/index.js",
   "dependencies": {
-    "json5": "^2.2.1",
-    "typescript": "^4.8.4"
+    "json5": "^2.2.1"
+  },
+  "scripts": {
+    "build": "tsc",
+    "prepublishOnly": "npm run build"
```

Comment:
that is difference between prepublish and prepublishOnly?

Reply:
https://docs.npmjs.com/cli/v8/using-npm/scripts  seems like it is automatically run in during npm publish, in comparison with prepublish